massa-api/src/public.rs

```diff
@@ -479,17 +454,23 @@ impl Endpoints for API<Public> {
                 Map::with_capacity_and_hasher(addresses.len(), BuildMap::default());
             let mut endorsements: Map<Address, Set<EndorsementId>> =
                 Map::with_capacity_and_hasher(addresses.len(), BuildMap::default());
+            let mut final_sce_ledger_info: Map<Address, SCELedgerInfo> =
+                Map::with_capacity_and_hasher(addresses.len(), BuildMap::default());
+            let mut candidate_sce_ledger_info: Map<Address, SCELedgerInfo> =
+                Map::with_capacity_and_hasher(addresses.len(), BuildMap::default());
 
             let mut concurrent_getters = FuturesUnordered::new();
             for &address in addresses.iter() {
                 let mut pool_cmd_snd = pool_command_sender.clone();
                 let cmd_snd = cmd_sender.clone();
+                let exec_snd = execution_controller.clone();
```

Comment:
I'm skeptical about this clone you use it only to get a specific element could you just clone the element  L.493 ?

Reply:
the issue is that we need these clone because async move